### ISSUE TITLE
NOT WORK

### ISSUE THREAD

**Issue by dmitijmaslak** [spinza/tuya_mqtt#3]
Always have error
```
2024-03-21 02:27:49,836 - tuya_mqtt - INFO - Starting tuya_mqtt...
2024-03-21 02:27:49,837 - tuya_mqtt - ERROR - Device file not found.
(.venv) root@spruthub:/opt/tuya_mqtt# ls -al
total 88
drwxr-xr-x 4 root root  4096 Mar 21 02:10 .
drwxr-xr-x 5 root root  4096 Mar 21 01:55 ..
-rw-r--r-- 1 root root   989 Mar 21 01:55 config_defaults.py
-rw-r--r-- 1 root root   529 Mar 21 01:55 config.py
-rw-r--r-- 1 root root  2682 Mar 21 02:16 devices.json
-rw-r--r-- 1 root root  1223 Mar 21 01:55 .gitignore
drwxr-xr-x 2 root root  4096 Mar 21 01:58 __pycache__
-rw-r--r-- 1 root root  2018 Mar 21 01:55 README.md
-rw-r--r-- 1 root root   299 Mar 21 01:58 requirements.txt
-rwxr-xr-x 1 root root 28729 Mar 21 02:25 server.py
-rw-r--r-- 1 root root   938 Mar 21 02:16 snapshot.json
-rw-r--r-- 1 root root   161 Mar 21 01:55 tinytuya.json
-rw-r--r-- 1 root root   499 Mar 21 01:55 tuya_mqtt.service
-rw-r--r-- 1 root root  2116 Mar 21 02:06 tuya-raw.json
drwxr-xr-x 7 root root  4096 Mar 21 01:55 .venv
(.venv) root@spruthub:/opt/tuya_mqtt# ./server.py
2024-03-21 02:27:53,416 - tuya_mqtt - INFO - Starting tuya_mqtt...
2024-03-21 02:27:53,417 - tuya_mqtt - ERROR - Device file not found.
```

**Comment by psmedley** [comment]
me too. Seems DEVICE_FILE in config.py needs to be TINYTUYA_DEVICE_FILE

**Comment by spinza** [comment]
Fixed by e84e7037b57849ee56ab4422e1c2fd9664905e06

DEVICE_FILE is now the correct way.
